packages/admin-ui/src/pages/packages/components/Info/VolumesList.tsx

```diff
@@ -79,6 +79,8 @@ export const VolumesList = ({ dnp }: { dnp: InstalledPackageDetailData }) => {
     return null;
   }
 
+  const isTotalVolumeSizeIllDefined = volumes.map(v => v.size).includes(undefined);
```

Comment:
Is the word `isTotalVolumeSizeIllDefined` a typo? what does it mean ...size ill defined ?

Reply:
Ill defined means not well defined. But we can change that in something more readable.